src/main/python/hydra/lib/hydrabase.py

```diff
@@ -568,7 +554,7 @@ def reset_all_app_stats(self, name, group_name=""):
             ha_sub.reset_stats()
             ha_sub.stop()  # closes the ANalyser socket, can not be used anymore
 
-    def ping_all_app_inst(self, name, group_name=""):
+    def remove_unresponsive_tasks(self, name, group_name=""):
```

Comment:
we should split the functions in my opinion, ping_all_app is just pinging apps, the previous function was implicitly removing app tasks as well.

I think ping_all_app is still a useful function and can be kept separate.
Also unless your hydra user frameworks are updated, the pipelines using hydra will fail because of name change. So both changes need to go in together.

Generally we should avoid renaming API unless strictly necesary, adding new ones are fine.